src/components/layout/nav/components/markets/nav-market-mobile.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { ReactElement } from 'react'
+import styled from 'styled-components'
+import { NavLink, StyledLinkMarket, MobileWrapper, Wrapper } from '../../styles/nav-styles'
+import { localize } from 'components/localization'
+import { useActiveLinkState } from 'components/hooks/use-active-link-state'
+
+const StyledWrapper = styled(Wrapper)`
+    justify-content: center;
+`
+const NavigationBar = styled.ul`
+    display: flex;
+    justify-content: center;
+    padding: 0;
+    line-height: 1.2;
+    margin-left: 13.3rem;
+    white-space: nowrap;
+
+    @media (max-width: 1300px) {
+        font-size: 12px;
+    }
+`
+type NavLinkCardTypes = {
+    to: string
+    title: ReactElement
+    active?: string
+    type?: string
+    target?: string
+    external?: string
+    rel?: string
+}
+
+const NavLinkCard = ({ title, active, ...rest }: NavLinkCardTypes) => {
+    const current_page = useActiveLinkState('markets')
+    return (
+        <NavLink>
+            <StyledLinkMarket
+                active={current_page === active}
+                activeClassName="active"
+                aria-label={title}
+                {...rest}
+            >
+                {title}
+            </StyledLinkMarket>
+        </NavLink>
+    )
+}
+
+const NavMarketMobile = () => {
+    return (
+        <MobileWrapper>
```

Comment:
If the purpose of this wrapper is to hide this on desktop then we have `<Mobile>` component on `visibility.tsx` we can use that instead

Reply:
Hi @prince-deriv , its a styled component for mobile wrapper.